forc-plugins/forc-debug/src/server/util.rs

```diff
@@ -0,0 +1,39 @@
+use dap::types::Source;
+use fuel_vm::fuel_asm::RegId;
+use std::path::Path;
+
+#[derive(Debug, Clone)]
+pub(crate) struct IdGenerator {
```

Comment:
Might also be good to add a doc comment explaining what an IdGenerator is and why it's needed